phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/ViewInfoTracker.java

```diff
@@ -24,24 +24,6 @@
 import java.io.IOException;
 
 public class ViewInfoTracker implements ViewInfoWritable {
```

Comment:
nit: Good job on refactoring out it to an interface. Now that you have refactored it out it can be called ViewTTLInfoWritable? Since it contains TTL related info